engine/src/flutter/lib/ui/platform_dispatcher.dart

```diff
@@ -3189,3 +3211,22 @@ enum ViewFocusDirection {
   /// This is typically result of the user pressing shift + tab.
   backward,
 }
+
+/// The request containing all the information required to perform framework hitTest.
+class HitTestRequest {
+  const HitTestRequest({required this.view, required this.offset});
+
+  /// The flutter view.
```

Comment:
This comment is not useful, since the document is for implementing the callback of PlatformDispatcher.onHitTest, should talk about how this property should be used